### DISCUSSION TITLE
SIB12 Paging

### DISCUSSION THREAD

**Discussion by parv125** [srsran/srsRAN_Project#53] [General Help]
I was recently tasked with demonstrating use cases for a few projects at work and I'm attempting to deliver SIB12 with CMAS alert. I've edited the parser and added other SIB's and they've worked fine but for SIB12 I can't seem to get the CMAS message to show on the UE itself. When I capture the eNodeB transmission via wireshark it shows everything intact but even when the UE attempts to connect it doesn't show the alert. My assumption is paging is not needed as it should decode all SIB's when it attempts cell reselection, but I could definitely be wrong, do I need to initiate paging as well even if the phone is attempting to attach to my tower? I attached a .pcap and configs of the SIB12 broadcast if anyone can point me in the right direction. SIB_12_UE_nonreact.txt is a pcap but it wouldn't let me attach the file without naming it .txt.
[sib.conf.txt](https://github.com/srsran/srsRAN_Project/files/11217551/sib.conf.txt)
[rr.conf.txt](https://github.com/srsran/srsRAN_Project/files/11217552/rr.conf.txt)
[enb.conf.txt](https://github.com/srsran/srsRAN_Project/files/11217553/enb.conf.txt)

[SIB12_UE_nonreact.txt](https://github.com/srsran/srsRAN_Project/files/11217546/SIB12_UE_nonreact.txt)

**Comment by andrepuschmann** [comment]
Wrong repo, the 4G one is here https://github.com/srsran/srsRAN_4G

**Comment by stackprogramer** [comment]
@parv125  Hi, can you solve your problem, I wan to do the same work can you guide me? thanks in advance
